### PR TITLE
 Consistent Formatting in README Markdown File

### PR DESCRIPTION
**Pull request by ankman007**
### Issue Description
Correctly formatted README.md file with consistent formatting and linting. Also, included `<hr/>` prior to H3. 

Solved Issue #11 

### Screenshots

readme
_Screenshot 1_

### CODE REVIEW

**Comment by ankman007** [comment]
@gonX Do let me know if I need to make few more changes to markdown before merge.

**Comment by X9VoiD** [comment]
There's no need for the `<hr />`s in README.md. The linked issue is only concerned on adding them in md files existing inside `_wiki`.

**Review by gonX — APPROVED** [review]
Looks good, though the change does not help with #11 since that is specifically for [linting](https://en.wikipedia.org/wiki/Lint_(software)).

Linting Markdown files would be accomplished using something like [markdownlint](https://github.com/markdownlint/markdownlint) and connecting it to a GitHub actions similar to [Lint Action](https://github.com/marketplace/actions/lint-action) (which does not support Markdown).

Regardless, fixing up the README is very appreciated, it was rather rushed on my end.

**Comment by gonX** [comment]
> There's no need for the `<hr />`s in README.md.

The other changes are nice, and breaks do help with readability - we can always change that part back if it looks messy.

File: README.md [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
-## Requirements
+## Local Development
+
+### Requirements
 
 - `rubygems`
 
+---
+
 ## Steps
```
Comment (gonX) [inline README.md:14]:
```suggestion
## Steps
```

only before H3's (or in the case of the readme, H1's)

**Pull request merged**